### ISSUE TITLE
Update grass-70 to rc1

### ISSUE THREAD

**Issue by rkrug** [OSGeo/homebrew-osgeo4mac#74]
Hi

I changed the grass-70 formula to rc-1.

The changes are minimal, only requiring the change of the source.
```
url "http://grass.osgeo.org/grass70/source/grass-7.0.0RC1.tar.gz"
sha1 "316d638ccf4f7dfb588f7e4e69243c18cd18bd70"
version "7.0.0RC1"
```

The installation worked without problems.
Hope this can be changed as quite a few things have changed between the betas and the RC.

Cheers,

Rainer

**Comment by dakcarto** [comment]
Hi Rainer,

Thanks for reporting.
Should be fixed in e97f7b9. Reopen if necessary.

**Comment by rkrug** [comment]
Thanks

Rainer
